autopush/db.py

```diff
@@ -464,7 +464,7 @@ def __init__(self, **kwargs):
         if "endpoint_url" in conf and not conf["endpoint_url"]:
             del(conf["endpoint_url"])
         region = conf.get("region_name",
-                          os.getenv("AWS_REGION_NAME", "us-east-1"))
+                          os.getenv("AWS_DEFAULT_REGION", "us-east-1"))
```

Comment:
we might be able to nuke this entirely (just let boto3 look it up) but we can do this later

Reply:
Agreed.